Resources/Textures/Objects/Devices/communication.rsi/meta.json

```diff
@@ -0,0 +1,183 @@
+{
+  "version": 1,
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "license": "CCBYNA",
```

Comment:
Needs to be a valid spdx format, also I would've assume it's the same licence as the rest but I wouldn't know.